client/src/pages/Feed.js

```diff
@@ -18,21 +30,151 @@ import {
 // ICONS
 import SvgIcon from "components/Icon/SvgIcon";
 import creatPost from "assets/icons/create-post.svg";
+import { ReactComponent as FiltersIcon } from "assets/icons/filters.svg";
 
 export const FeedContext = React.createContext();
 
+const { Content, Sider } = Layout;
+
+// feed types
+const HELP_TYPE = {
+  ALL: 'All posts',
+  REQUEST: 'Requesting help',
+  OFFER: 'Offering help'
+};
+
 const initialState = {
   filterModal: false,
   createPostModal: false,
   activePanel: null,
   location: "",
 };
 
+const SiderWrapper = styled(Sider)`
+  background-color: ${WHITE};
+  height: calc(100vh - 60px);
+  overflow-x: hidden;
+  padding-top: 33px;
+  position: fixed;
+
+  @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
+    display: none;
+  }
+`;
+
+const FiltersWrapper = styled.div`
+  border-top: 0.5px solid rgba(0, 0, 0, 0.5);
+  margin: 0 20px;
+  padding-top: 20px;
+
+  button {
+    align-items: center;
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+    display: flex;
+    font-family: ${theme.typography.font.family.display};
+    font-size: ${theme.typography.size.large};
+    font-weight: bold;
+    margin-bottom: 10px;
+    padding: 0;
+
+    span {
```

Comment:
Let's add a hover color of `ROYAL_BLUE`

current
<img width="181" alt="Screen Shot 2020-05-10 at 7 50 55 PM" src="https://user-images.githubusercontent.com/42228386/81521542-6872f680-92fc-11ea-9fd6-008427f5eb98.png">

Reply:
Done. Used DARKER_GRAY - #282828.